packages/slice-machine/src/modules/selectedSlice/reducer.ts

```diff
@@ -261,13 +261,16 @@ const updateTouchedAndStatus = (model: SliceState) => {
     model.initialVariations
   );
 
-  if (isModelModified || isScreenshotModified) {
+  if (
```

Comment:
damn all those conditions seems too though to read, i'm sure we could simplify that later :)